tests/kernel/mem_protect/demand_paging/testcase.yaml

```diff
@@ -2,3 +2,13 @@ tests:
   kernel.memory_protection.demand_paging:
     tags: kernel mmu demand_paging ignore_faults
     filter: CONFIG_DEMAND_PAGING
+  kernel.memory_protection.demand_paging.qemu_x86.timing_funcs:
```

Comment:
this is getting too long, maybe just `kernel.demand_paging.statistics`?, same for the existing one, `kernel.demand_paging`. Do not see a need for the platform name here.

Reply:
Made the name shorter.